test/unit/components/sound-editor.test.jsx

```diff
@@ -6,6 +6,8 @@ describe('Sound Editor Component', () => {
     let props;
     beforeEach(() => {
         props = {
+            autoCapitalize: 'none',
```

Comment:
Is this needed?

Reply:
you are right, I have removed this line.